tests/buyers/views/test_buyers.py

```diff
@@ -22,38 +22,115 @@
 po = functools.partial(mock.patch.object, autospec=True)
 
 
+@pytest.fixture()
+def find_briefs_mock():
+    base_brief_values = {
+        "createdAt": "2016-02-01T00:00:00.000000Z",
+        "frameworkSlug": "digital-outcomes-and-specialists",
+        "lot": "digital-specialists"
+    }
+
+    find_briefs_response = {
+        "briefs": [
+            {
+                "id": 20,
+                "status": "draft",
+                "title": "A draft brief",
+            }, {
+                "id": 21,
+                "status": "live",
+                "title": "A live brief",
+                "publishedAt": "2016-02-04T12:00:00.000000Z",
+            }, {
+                "id": 22,
+                "status": "closed",
+                "title": "A closed brief",
+                "publishedAt": "2016-02-04T12:00:00.000000Z",
+                "applicationsClosedAt": "2016-02-18T12:00:00.000000Z",
+            }, {
+                "id": 23,
+                "status": "withdrawn",
+                "title": "A withdrawn brief",
+                "publishedAt": "2016-02-04T12:00:00.000000Z",
+                "withdrawnAt": "2016-02-05T12:00:00.000000Z",
+            }
+        ]
+    }
+
+    for brief in find_briefs_response['briefs']:
+        brief.update(base_brief_values)
+
+    return find_briefs_response
+
+
 @mock.patch('app.buyers.views.buyers.data_api_client')
 class TestBuyerDashboard(BaseApplicationTest):
-    def test_buyer_dashboard(self, data_api_client):
-        with self.app.app_context():
-            self.login_as_buyer()
-            data_api_client.find_briefs.return_value = {
-                "briefs": [
-                    {"status": "draft",
-                     "title": "A draft brief",
-                     "createdAt": "2016-02-02T00:00:00.000000Z",
-                     "frameworkSlug": "digital-outcomes-and-specialists"},
-                    {"status": "live",
-                     "title": "A live brief",
-                     "createdAt": "2016-02-01T00:00:00.000000Z",
-                     "publishedAt": "2016-02-04T12:00:00.000000Z",
-                     "frameworkSlug": "digital-outcomes-and-specialists"},
-                ]
-            }
 
-            res = self.client.get("/buyers")
-            document = html.fromstring(res.get_data(as_text=True))
+    def setup_method(self, method):
+        super(TestBuyerDashboard, self).setup_method(method)
+        self.login_as_buyer()
 
-            assert res.status_code == 200
+    def test_draft_briefs_section(self, data_api_client, find_briefs_mock):
+        data_api_client.find_briefs.return_value = find_briefs_mock
```

Comment:
This test class is nice.
The first 3 lines on every test are being repeated but up to you if you want to try and move that into the setup or not